client/src/App.js

```diff
@@ -1,11 +1,20 @@
 import "./App.css";
+import InsertVideo from "./InsertVideo";
+// import video from "./singleVideo.json";
+import defaultVideos from "./exampleresponse.json";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <header className="my-App-header">
         <h1>Video Recommendation</h1>
       </header>
+      <body>
+          {defaultVideos.map((video, key) => (
```

Comment:
Rendering the data you imported directly on your page is not a good idea as you will need to manipulate/update it at some point based on the user interaction. Storing the data in a react state variable is your best bet.

Reply:
I have completed level 100. Can you please review it? Thanks